modules/core/src/lib/attribute.js

```diff
@@ -45,17 +45,25 @@ export default class Attribute extends BaseAttribute {
         // Initialize the attribute descriptor, with WebGL and metadata fields
         this.shaderAttributes[shaderAttributeName] = new Attribute(
           this.gl,
-          Object.assign({}, shaderAttribute, {
-            id: shaderAttributeName,
-            // Luma fields
-            constant: shaderAttribute.constant || false,
-            isIndexed: shaderAttribute.isIndexed || shaderAttribute.elements,
-            size: (shaderAttribute.elements && 1) || shaderAttribute.size || this.size,
-            value: shaderAttribute.value || null,
-            divisor: shaderAttribute.instanced || shaderAttribute.divisor || this.divisor,
-            buffer: this.getBuffer(),
-            noAlloc: true
-          })
+          Object.assign(
+            {
+              offset: this.offset,
+              stride: this.stride,
+              normalized: this.normalized
```

Comment:
the `normalized` flag is kind of confusing, from the name looks like the attribute is already normalized. But actually, with this flag on the data need normalization, not normalized yet.

Reply:
This is a WebGL flag https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/vertexAttribPointer